templates/_base/master/units/kubelet.yaml

```diff
@@ -0,0 +1,65 @@
+name: "kubelet.service"
+enabled: true
+contents: |
+  [Unit]
+  Description=Kubernetes Kubelet
+  Wants=rpc-statd.service
+  Requires=docker.service kubelet-workaround.service
```

Comment:
This makes sense from a porting perspective, but I fear this will be problematic with 2 kubelets available on the base OS (one containerized, one installed).

Reply:
Reference: https://github.com/openshift/os/issues/170

Reply:
These templates can and will be changed in future to match RHCOS.